custom/icds_reports/static/js/directives/download/download.directive.js

```diff
@@ -104,6 +104,7 @@ function DownloadController($rootScope, $location, locationHierarchy, locationsS
     vm.selectedPDFFormat = 'many';
     vm.selectedLocationId = userLocationId;
     vm.selectedLevel = 1;
+    vm.selectedLocationLevel = 0;
```

Comment:
why do we need both selectedLocationLevel and selectedLevel? and how are they different? and I guess also how is that different than determining that level via the selectedLocations array

Reply:
Hey, good question. Actually the difference is selected level was being manipulated by location filter as well as the view by filter. `selectedLocationLevel` I am just manipulating by location filters. Yeah, I agree it could also be pulled from selectedLocationsarray. I actually was doing that in the first try. But, I think using a separate variable is cleaner than that.  Feel free to disagree if you feel the opposite is